src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.json

```diff
@@ -1,5 +1,5 @@
 {
-  "Name": "Microsoft.Extensions.AI.Abstractions, Version=9.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35",
+  "Name": "Microsoft.Extensions.AI.Abstractions, Version=9.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35",
```

Comment:
Is this necessary?

Reply:
No, was changed automatically by `MakeApiBaselines.ps1`, will undo.